packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -129,10 +131,12 @@ export class Outbox {
             "BatchTooLarge",
             /* error */ undefined,
             {
-                opSize: batch.contentSizeInBytes,
-                count: batch.content.length,
+                batchSize: batch.contentSizeInBytes,
+                compressedBatchSize: compressedBatch.contentSizeInBytes,
+                count: compressedBatch.content.length,
```

Comment:
I'm assuming this is the number of ops in the batch? And that should match for the `compressedBatch` and `batch`?

Reply:
spot on. Yes, they should match.